imgee/views/files.py

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8 -*-
+from flask import (flash, g, jsonify, redirect, render_template, request, url_for)
+from sqlalchemy import and_
+
+from coaster.views import load_model, load_models
+from imgee import app, forms, lastuser
```

Comment:
We typically use `forms` for `from baseframe import forms`, so it may be confusing to someone reading this file and wondering what the `forms` namespace is.